chapters/annotations.tex

```diff
@@ -888,6 +888,11 @@ \subsection{Component Instance}\label{component-instance}
 Sub-connectors in a hierarchical connector are only shown when they can be connected to.
 Non-connector components are only shown in the diagram layer.
 
+\subsubsection{Default Outline for Missing Graphics}\label{default-outline-for-missing-graphics}
+If the icon of a class does not contain any graphical primitives (including inherited ones, and regardless of visibility; but excluding connectors) it is said to be missing graphics.
+If the icon of a class used in a diagram is missing graphics then tools shall show a tool-dependent outline of the extent of the component to improve usability.
+The model with missing graphics should be updated if its icon is used in any diagram.
```

Comment:
First it was the icon that was missing graphics.  Here it is the model.

I'm not sure how much we can state in the specification regarding the need to have an icon, especially considering that the library developer generally doesn't know how a model is used outside of the library.

Can't we instead say something based on the presence of connectors with placement on the icon?

Reply:
Given the suggested non-normative explanation above, I suggest simply removing this sentence:
```suggestion
```